recipes/header/settings_menu_button/settings_menu_button_variants.story.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
After coding this up I'm not sure that this recipe really has any variants? Left this here just in case but am happy to delete it.